sources/core/objects/Clothes.h

```diff
@@ -73,6 +73,14 @@ class OrangeBoots: public Cloth
     OrangeBoots(quint32 id);
 };
 ADD_TO_TYPELIST(OrangeBoots);
+class ClownBoots: public Cloth
```

Comment:
Add 1 empty line after `ADD_TO_TYPELIST(OrangeBoots);` - it looks better that way.